app/scripts/lib/AddressBookPetnamesBridge.ts

```diff
@@ -11,7 +11,10 @@ import {
   PetnameEntry,
 } from './AbstractPetnamesBridge';
 
-export class AddressBookPetnamesBridge extends AbstractPetnamesBridge {
+export class AddressBookPetnamesBridge extends AbstractPetnamesBridge<
+  never,
```

Comment:
Can we remove these since we added the default templates?

Reply:
Yes we can remove these